manifests/07_deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aws-ebs-csi-driver-operator
+  namespace: openshift-cluster-csi-drivers
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: aws-ebs-csi-driver-operator
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        name: aws-ebs-csi-driver-operator
+    spec:
+      containers:
+      - args:
+        - start
+        env:
+        - name: DRIVER_IMAGE
+          value: quay.io/openshift/origin-aws-ebs-csi-driver:latest
```

Comment:
Not necessarily a bug in this PR, but how we're going to copy this manifest to CSO? We need to add `${DRIVER_IMAGE}`, so CSO can replace it with the current driver image. Can a script in CSO be good enough?
[applies to all images below]

Reply:
Do we even need to carry these manifests in this repository? if CSO is going to install them and there is no programming way for CSO to get them, shouldn't this directly live in CSO repo?

Reply:
Good point... Who is going to use manifests from this repo? The e2e test will be obsolete as soon as CSO installs the operator by default.

Reply:
I'd propose to remove the manifests together with e2e test *after* CSO is updated.

Reply:
We discussed it on a meeting today and we decided to keep the manifests on both places, here and in CSO. We will try to find a better way how to sync the manifests automatically.